codecept.conf.js

```diff
@@ -49,21 +66,31 @@ exports.config = {
     ordersNeededPage: './e2e/pages/ordersNeeded/ordersNeeded.js',
     enterFamilyManPage: './e2e/pages/enterFamilyMan/enterFamilyMan.js',
     changeCaseNamePage: './e2e/pages/changeCaseName/changeCaseName.js',
-    submitApplicationPage: './e2e/pages/submitApplication/submitApplication.js'
+    submitApplicationPage: './e2e/pages/submitApplication/submitApplication.js',
   },
   plugins: {
     autoDelay: {
       enabled: true,
+      methods: [
+        'click',
+        'doubleClick',
+        'rightClick',
+        'fillField',
+        'pressKey',
+        'checkOption',
+        'selectOption',
```

Comment:
This adds `selectOption` operation to list of auto delayed operations. It will add 100ms delay before the call and 200ms delay after.